src/ORM/Table.php

```diff
@@ -123,26 +124,26 @@
  *
  * @see \Cake\Event\EventManager for reference on the events system.
  */
-class Table implements RepositoryInterface, EventListenerInterface, EventDispatcherInterface
+class Table implements RepositoryInterface, EventListenerInterface, EventDispatcherInterface, ValidatorAwareInterface
 {
 
     use EventDispatcherTrait;
     use RulesAwareTrait;
     use ValidatorAwareTrait;
 
     /**
-     * Name of default validation set.
+     * The alias this object is assigned to validators as.
      *
      * @var string
      */
-    const DEFAULT_VALIDATOR = 'default';
+    const VALIDATOR_PROVIDER_NAME = 'table';
 
     /**
-     * The alias this object is assigned to validators as.
+     * The name of the event dispatched when a validator has been built.
      *
      * @var string
      */
-    const VALIDATOR_PROVIDER_NAME = 'table';
+    const BUILD_VALIDATOR_EVENT = 'Model.buildValidator';
```

Comment:
Here in a trait: https://github.com/cakephp/cakephp/pull/10707/files#diff-5ea28017bc64b94c8fdbe87e3b422986R178

I wanted this trait to be used outside model scope.